### PR TITLE
Freewheel: add video dur and cat

### PR DESCRIPTION
**Pull request by mwang-sticky**

### CODE REVIEW

File: adapters/freewheelssp/freewheelssp.go [hunk 1]
```diff
@@ -87,18 +87,27 @@ func (a *adapter) MakeBids(internalRequest *openrtb2.BidRequest, externalRequest
...
+			}
+			bidResponse.Bids = append(bidResponse.Bids, adTypeBid)
 		}
 	}
 	return bidResponse, nil
 }
 
 func Builder(bidderName openrtb_ext.BidderName, config config.Adapter, server config.Server) (adapters.Bidder, error) {
-	bidder := &adapter{
+	bidder := &Adapter{
```
[the remark this answers is not in the document]
Comment (mwang-sticky) [reply adapters/freewheelssp/freewheelssp.go:110]: Thanks, I simplify the name

File: adapters/freewheelssp/freewheelssp.go [hunk 2]
```diff
@@ -12,11 +12,11 @@ import (
 	"github.com/prebid/prebid-server/v2/openrtb_ext"
 )
 
-type adapter struct {
-	endpoint string
+type Adapter struct {
```
Comment (gargcreation1992) [inline adapters/freewheelssp/freewheelssp.go:15]: This does not have to be an exported field. Please change it to use lowercase `type adapter struct`

**Review by gargcreation1992 — DISMISSED** [review]

File: static/bidder-info/default.profraw
Comment (Sonali-More-Xandr) [inline static/bidder-info/default.profraw:1]: Can you please explain why is this file required? Please remove this file if not needed

File: adapters/freewheelssp/freewheelssp_test.go [hunk 3]
```diff
@@ -18,3 +22,79 @@ func TestJsonSamples(t *testing.T) {
 
 	adapterstest.RunJSONBidderTest(t, "freewheelssptest", bidder)
 }
+
+func TestFreewheelAdapter_MakeBids(t *testing.T) {
```
Comment (gargcreation1992) [inline adapters/freewheelssp/freewheelssp_test.go:26]: This is not how you should test your code. Please create json test cases. Refer - https://github.com/prebid/prebid-server/blob/2cc2c90a09708de6fe5cdbf5d798a08db0b806d9/adapters/freewheelssp/freewheelssptest/exemplary/multi-imp.json#L3

**Comment by gargcreation1992** [comment]
@mwang-sticky PR checks are failing because of - `expected.ImpIDs must contain at least one imp ID`

**Comment by mwang-sticky** [comment]
> @mwang-sticky PR checks are failing because of - `expected.ImpIDs must contain at least one imp ID`

https://github.com/prebid/prebid-server/commit/30082c04295370d46d57db4526c80be8f52806f2#diff-d83f728c5918299ec6f5a8d22447adb9fefac2726ab505dea12ba609dd79e068 ?

**Review by gargcreation1992 — APPROVED** [review]

**Review by Sonali-More-Xandr — APPROVED** [review]
